src/Ocelot/Configuration/Builder/AuthenticationOptionsBuilder.cs

```diff
@@ -7,6 +7,8 @@ public class AuthenticationOptionsBuilder
         private List<string> _allowedScopes = new();
         private string _authenticationProviderKey;
 
+        private List<string> _authenticationProviderKeys = new();

```

Comment:
maybe, it could be a real array

Reply:
✔️ Fixed in commit https://github.com/ThreeMammals/Ocelot/pull/1666/commits/10343db6558a697586e880f486e34909fdf1cd10